oml/lightning/entrypoints/train.py

```diff
@@ -134,6 +134,7 @@ def pl_train(cfg: TCfg) -> None:
         logger=logger,
         precision=cfg.get("precision", 32),
         **trainer_engine_params,
+        **cfg.get("extra_args_for_lightning_trainer", {}),
```

Comment:
Following the other namings in Config-API I'd suggest to call it `lightning_trainer_extra_args`.

Reply:
okay